### ISSUE TITLE
Request: Show which version of a mod I've downloaded

### ISSUE THREAD

**Issue by fadingsignal** [Nexus-Mods/web-issues#1086]
Simple request: The ability to see which version of a mod I've downloaded.

Currently, when visiting a mod page, one will see "You last downloaded a file from this mod on 27 Jul 2019" as opposed to the version.  However, when browsing comments on one's page as an author, we have the ability to view a user's download history under Manage, which shows more detail: "29 Sep 2019, 5:20PM True Storms Special Edition v1.01, version 1.01"

tl;dr I would love to see the same data from Manage > Download History show on the mod page where it currently just shows the date

It could be as simple UI-wise as tacking the version onto the end of what's already there, like this:

![image](https://user-images.githubusercontent.com/13229928/71862669-7c3cf080-30af-11ea-9ed1-d1e51262d927.png)

Just a humble request. Thank you!

**Comment by Pickysaurus** [comment]
Not a bad idea, but if the mod files are not versioned properly this wouldn't be super useful. I'll keep a note for when the update mod pages though :)

**Comment by jarrard** [comment]
Perhaps just list the filename also, in case versioning is not updated correctly.

**Comment by Darklocq** [comment]
+1.  Please do not reject the feature req. on the basis that some mod uploaders don't do versioning well. If that were a real issue, it would be a rationale to delete the same feature from Download History under Manage, but that feature is useful.  I'm neutral on whether to show the filename.

**Comment by Pickysaurus** [comment]
Each individual file now has the date/time of last download displayed on it.
